nsds_lab_to_nwb/common/time.py

```diff
@@ -0,0 +1,21 @@
+import logging.config
```

Comment:
Is it good practice to name a file 'time' since there's a common python package called 'time'? Functiionally it should be ok because the package import will be specific to this nsds package. But just wondering

Reply:
Thanks John for the review! I'm also not completely sure but my feeling is that since the module is specified by the entire package hierarchy (like `nsds_lab_to_nwb.common.time`) the name is distinct enough…? I'll merge this for now, but please let me know if you find something that advices otherwise!